packages/neon-core/src/u/basic/hex.ts

```diff
@@ -1,5 +1,16 @@
 const hexRegex = /^([0-9A-Fa-f]{2})*$/;
 
+/**
+ * Format a hexstring to a supported format.
+ */
+export function formatHex(str: string): string {
```

Comment:
I think this name is too generic and doesn't actually describes what the method is doing. I think it could be named `remove0xPrefix`, `strip0xPrefix`, `removeLeading0x`, something like this.